include/nbl/asset/utils/ISPIRVDebloater.h

```diff
@@ -0,0 +1,40 @@
+#ifndef _NBL_ASSET_I_SPIRV_DEBLOATER_H_INCLUDED_
+#define _NBL_ASSET_I_SPIRV_DEBLOATER_H_INCLUDED_
+
+#include "nbl/core/declarations.h"
+
+#include "nbl/asset/ICPUBuffer.h"
+
+#include "nbl/system/ILogger.h"
+
+namespace nbl::asset
+{
+
+class ISPIRVDebloater final : public core::IReferenceCounted
+{
+    public:
+        ISPIRVDebloater();
+
+        struct Result
+        {
+            bool isAllEntryPointsFound;
```

Comment:
slight grammar nitpick, multiple is either `are` or `were`

Also another nitpick, make sure to order your member fields from largest to smallest (best packing with alignment padding). Right now you'll get same size either way but its a good habit to get into